services/onboarding-service/src/main/java/com/amazon/aws/partners/saasfactory/saasboost/OnboardingService.java

```diff
@@ -1029,6 +1029,10 @@ protected void handleOnboardingBaseProvisioned(Map<String, Object> event, Contex
                         templateParameters.add(Parameter.builder().parameterKey("RDSPort").parameterValue(dbPort.toString()).build());
                         templateParameters.add(Parameter.builder().parameterKey("RDSDatabase").parameterValue(dbDatabase).build());
                         templateParameters.add(Parameter.builder().parameterKey("RDSBootstrap").parameterValue(dbBootstrap).build());
+                        templateParameters.add(Parameter.builder()
+                                .parameterKey("S3ExtensionBucket")
+                                .parameterValue(getSetting(context, "EXTENSION_BUCKET"))
```

Comment:
If we change the same to storage or objects or something else, we should be consistent throughout and change it in param store and wherever else